pkg/controller/map/network/validation.go

```diff
@@ -147,19 +146,6 @@ func (r *Reconciler) validateDestination(mp *api.NetworkMap) (err error) {
 	}
 	list := mp.Spec.Map
 	notFound := []string{}
-	var resource interface{}
-	switch provider.Type() {
-	case api.OpenShift:
-		resource = &ocp.NetworkAttachmentDefinition{}
-	case api.VSphere:
-		return
-	default:
-		err = liberr.Wrap(
-			web.ProviderNotSupportedError{
-				Provider: provider,
-			})
-		return
-	}
```

Comment:
Resource no longer needed because using inventory.Network() instead of inventory.Get().